app/assets/stylesheets/_sidebar.scss

```diff
@@ -1,77 +0,0 @@
-.sidebar-container {
```

Comment:
I removed this file and moved all of its contents to either `organizations.scss` or `layout.scss`. There's really not much in here that deals with the sidebar itself.